implementations/rust/ockam/ockam_api/src/cloud/project.rs

```diff
@@ -211,7 +211,8 @@ pub struct CreateProject<'a> {
     #[b(1)] pub name: CowStr<'a>,
     #[b(2)] pub services: Vec<CowStr<'a>>,
     #[b(3)] pub users: Vec<CowStr<'a>>,
-    #[b(4)] pub enforce_credentials: Option<bool>
+    #[b(4)] pub enforce_credentials: Option<bool>,
+    #[b(5)] pub identity_name: Option<String>
```

Comment:
You can use a `CowStr<'a>` instead of a `String` here. Sorry for not specifying this correctly in the issue description

Reply:
No worries, fixing it in the upcoming changes!

Reply:
Should be solved in the latest commit